runtime/src/in_mem_accounts_index.rs

```diff
@@ -264,6 +271,42 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
         result
     }
 
+    pub(crate) fn flush(&self) {
+        let flushing = self.flushing_active.swap(true, Ordering::Relaxed);
```

Comment:
Since `flushing_active` is like a thread-safe flag, and this part raises the flag, I think this Ordering should be `Ordering::Acquire`. This ensures no loads/stores get reordered _before_ the load part of this swap. I.e. nothing happens before saying `start flushing`.

Reply:
done